Mistica/Source/Components/Callout/Model/CalloutAsset.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  CalloutAsset.swift
+//
+//  Made with ❤️ by Novum
+//
+//  Copyright © Telefonica. All rights reserved.
+//
+
+import UIKit
+
+private enum Constants {
+    static let iconSize = CGFloat(24)
+    static let iconOffset = CGFloat(5)
```

Comment:
Same, although I don't know if we are using this convention in Mistica:

```suggestion
    static let iconSize: CGFloat = 24
    static let iconOffset: CGFloat = 5
```